src/java/org/apache/cassandra/index/sai/disk/v1/TermsReader.java

```diff
@@ -117,9 +118,14 @@ public void close()
     }
 
     public TermsIterator allTerms(long segmentOffset)
+    {
+        return allTerms(segmentOffset, true);
+    }
+
+    public TermsIterator allTerms(long segmentOffset, boolean ascending)
     {
         // blocking, since we use it only for segment merging for now
```

Comment:
@pkolaczk - is this a problem for us? I didn't look closely at what it meant by blocking.

Reply:
This code appears to be quite similar to other parts of the posting list logic, so I think we're fine here.